src/components/PivotTable/PivotTableValueCell.js

```diff
@@ -18,7 +18,7 @@ export const PivotTableValueCell = ({ row, column }) => {
     })
 
     if (!cellContent || cellContent.empty) {
-        return <PivotTableEmptyCell classes={cellContent?.cellType} />
```

Comment:
This is technically unrelated, but was causing a propTypes warning so I went ahead and fixed it